dali/kernels/signal/dct/dct_cpu.cc

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/kernels/signal/dct/dct_cpu.h"
+#include <cmath>
```

Comment:
Doesn't lint complain about this include being between internal includes?

Reply:
The header named exactly as .cc file (except extension) receives special treatment and is actually expected to come first. It's actually a good check that the header is self-contained.